src/main/scala/io/getclump/Clump.scala

```diff
@@ -52,12 +77,37 @@ object Clump extends Joins with Sources {
 
   def empty[T]: Clump[T] = value(scala.None)
 
+  /**
+   * Alias for [[value]]
+   */
+  def apply[T](value: T): Clump[T] = this.value(value)
```

Comment:
To keep a behavior similar to `Future.apply`[0], this method should propagate exceptions inside a Clump instance. Suggestion:

```scala
def apply[T](value: => T): Clump[T] = this.future(Future(value))
```

Same observation for the next method.

[0] https://github.com/scala/scala/blob/2.11.x/src/library/scala/concurrent/Future.scala#L492